src/daemons/rhsm_d.py

```diff
@@ -142,7 +143,7 @@ def __init__(self, bus, keep_alive, force_signal, loop):
         dbus.service.Object.__init__(self, name, "/EntitlementStatus")
         self.has_run = False
         #this will get set after first invocation
-        self.last_status = None
+        self.rhsm_icon_cache = require(RHSM_ICON_CACHE)
```

Comment:
@alikins ack, will do if we decide to use it only in update_status. per my earlier comment, thats TBD.